components/FAQ/molecules/Collage.tsx

```diff
@@ -0,0 +1,22 @@
+import Image from 'next/image';
+
+import FaqImage from 'public/assets/gyik.png';
+import {
+  CollageContainer,
+  LargerSquare,
+  SmallerSquare,
+  ImageContainer,
+} from 'components/FAQ/molecules/Collage.styles';
+
+const Collage = () => (
```

Comment:
Let me ask you a question, if these components in molecules are not using atoms, what makes them molecules?

Reply:
This question came up in me as well. I had 2 answers for myself :) 

First, I would argue that the inner implementation contains atoms even if they are not put into standalone files somewhere in a folder that is called 'atoms'.

Second, even if the above is not always the case, sometimes the size and/or complexity of the element just puts them into a bigger category in my head. I mean I could hardly consider them as 'atoms' when atoms are usually a header or a paragraph.

Even though I ended up using them this way, it is still a very thought-provoking question and the starter of an excellent conversation. I personally think that an analogy from chemistry should not induce overthinking or overly strict rules that do not necessarily help development. Meaning that the methodology could be fine in code, even if it breaks the rules and definition of stuff it took inspiration from.

Others, what's your opinion? @Roverr @orosznyet @balintant @mikitovicsmark

Reply:
Yeah I can completely agree with this argument 👍